201-web-app-python/metadata.json

```diff
@@ -4,7 +4,14 @@
   "itemDisplayName": "Create a web app on Azure with Python enabled",
   "description": "This template creates a web app on azure with Python enabled allowing you to run Python applications in Azure.  Template was authored by Donovan Brown of Microsoft.",
   "summary": "Create a web app on Azure with Python enabled.",
-  "githubUsername": "darquewarrior",
-  "dateUpdated": "2016-05-25"
+  "githubUsername": "leestott",
```

Comment:
do you want to take ownership of this one?

Reply:
yes happy to take over this from Donovan as it was well out of date and is a super popular workload for Azure4Students